.env

```diff
@@ -1,2 +1,3 @@
 REACT_APP_ENV
 REACT_APP_EMPLOYEES_URL
+REACT_APP_USER_AUTH_URL
```

Comment:
which should be the value of this new env? Could you update the readme with the values to run it locally?

Reply:
The config of those env vars are in the gdoc files